### PR TITLE
Update readme to begin flush out introduction and deployments

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,36 @@
-# falcon-helm
+# CrowdStrike Falcon Helm Chart and Helm Operator
+
+[Falcon](https://www.crowdstrike.com/) is the [CrowdStrike](https://www.crowdstrike.com/)
+platform purpose-built to stop breaches via a unified set of cloud-delivered
+technologies that prevent all types of attacks — including malware and much
+more.
```
Comment (shawndwells) [inline README.md:6]:
```suggestion
CrowdStrike's [Falcon Cloud Workload Protection](https://www.crowdstrike.com/cloud-security-products/falcon-cloud-workload-protection/)
provides complete visibility into cloud workload and container events and instance metadata enabling faster and more accurate detection, response, threat hunting and investigation, to ensure that nothing goes unseen in your cloud environment.
```

Comment (shawndwells) [reply README.md:6]: Taken from https://www.crowdstrike.com/cloud-security-products/falcon-cloud-workload-protection/

File: README.md [hunk 2]
```diff
@@ -1,7 +1,36 @@
...
+
+The CrowdStrike Falcon Helm Chart and the Helm Operator are designed to deploy
+and manage the Falcon sensor on your Kubernetes cluster of choice.
+[Helm Charts](https://helm.sh/docs/topics/charts/Helm) and Helm-based Kubernetes
+Operators are packaging methodologies for Kubernetes. Which methodology you use
+is dependent on what the Kubernetes vendor and implementation supports.
```
Comment (shawndwells) [inline README.md:12]:
```suggestion
The CrowdStrike Falcon Helm Chart and the Helm Operator are designed to deploy
and manage the Falcon sensor on your Kubernetes cluster of choice.

The Falcon platform supports Open
Container Initiative (OCI)-based containers such as Docker and Kubernetes and
also self-managed and hosted orchestration platforms such as GKE (Google
Kubernetes Engine), EKS (Amazon Elastic Kubernetes Service), ECS (Amazon
Elastic Container Service), AKS (Azure Kubernetes Service) and Red Hat's OpenShift.
```

Comment (shawndwells) [reply README.md:12]: Taken from bottom of page 4 https://www.crowdstrike.com/wp-content/uploads/2020/04/container-security-solution-brief.pdf

File: README.md [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
+
+# Deployment
```
Comment (shawndwells) [inline README.md:14]:
```suggestion
# Pre-launch Checklist
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,36 @@
...
+1. A Falcon sensor image must exist in your registry before deploying. See
+   https://github.com/CrowdStrike/Dockerfiles as an example of how to build a
+   Falcon sensor for your registry.
+1. The Falcon Linux (not the Falcon Container) should be used in the image to
+   deploy.
+1. Only deploying to Kubernetes Master and Worker nodes are supported at this
+   time.
+1. You need to have cluster admin privileges and be able to deploy daemonsets to
+   nodes
+1. The Helm Operator is under development and should not be used at this time.
```
Comment (shawndwells) [inline README.md:26]:
```suggestion
To ensure a successful deployment, you will want to ensure that:

1. You have access to a containerized falcon sensor image. This is most likely through a private image registry on your network or cloud provider. See https://github.com/CrowdStrike/Dockerfiles as an example of how to build a Falcon sensor for your registry.
  
1. You must have sufficient permissions to deploy Kubernetes DaemonSets. This is often received through cluster admin privileges.
   
1. CrowdStrike's Helm Operator is a project, not a product, and released to the community as a way to automate sensor deployment to kubernetes clusters. The upstream repository for this project is https://github.com/CrowdStrike/falcon-helm.
```

Comment (shawndwells) [reply README.md:26]: * What was meant by "the falcon Linux (not the falcon container) should be used"? Was not immediately clear. 

* In regards to deploying to kube master and worker nodes, are there any other types of nodes?

* Agree the "under development" should be in big red flashing letters. Good call! Should this be a top-level paragraph vs a bullet point users may overlook?

File: README.md [hunk 5]
```diff
@@ -1,7 +1,36 @@
...
 
 # Installation
 
+## Using the Helm Operator
+
+Currently under development
+
```
Comment (shawndwells) [inline README.md:33]: In reference to earlier comment, can put the "IN DEVELOPMENT NOT PRODUCTION READY" language here

File: README.md [hunk 6]
```diff
@@ -23,19 +52,17 @@ For example changing the default image repository using a yaml customization
...
-falcon:
-  cid: <Your_CrowdStrike_CID>
-image:
-  repository: 192.168.2.3:5000/falcon-sensor
-```
+   ```
+   falcon:
+     cid: <Your_CrowdStrike_CID>
+   image:
+     repository: 192.168.2.3:5000/falcon-sensor
```
Comment (shawndwells) [inline README.md:59]:
```suggestion
     repository: <Your_Registry>/falcon-sensor
```

**Review by shawndwells — CHANGES_REQUESTED** [review]

**Review by isimluk — APPROVED** [review]
👍 I have nothing to add.

File: README.md (hunk 3 above)
Comment (redhatrises) [reply README.md:14]: Let's make this `Deployment Considerations` instead of `Pre-launch Checklist`.

File: README.md (hunk 5 above)
Comment (redhatrises) [reply README.md:33]: Done

File: README.md (hunk 6 above)
Comment (redhatrises) [reply README.md:59]: Done

File: README.md (hunk 4 above)
Comment (redhatrises) [reply README.md]: @shawndwells made the updates. Also, made it clear that the "Falcon Linux Sensor RPM (not the Falcon Container) should be used in the container image to deploy to Kubernetes nodes."

File: README.md (hunk 3 above)
Comment (shawndwells) [reply README.md]: fiiiiiine

**Review by shawndwells — APPROVED** [review]

**Pull request merged**